openbb_terminal/cryptocurrency/defi/defipulse_model.py

```diff
@@ -12,6 +12,23 @@
 logger = logging.getLogger(__name__)
 
 
+@log_start_end(log=logger)
+def num_to_int(num):
+    if num[-1] == "M":
+        return int(float(num[1:-1])) * 1000000
+    if num[-1] == "B":
+        return int(float(num[1:-1])) * 1000000000
+    if num[-1] == "k":
+        return int(float(num[1:-1])) * 1000
+
+    return int(num)
+
+
+@log_start_end(log=logger)
```

Comment:
Same comments as above apply.

I would also avoid creating a function with 1 line of code. I would look into lambda to do that one liner function: 

e.g. `df['age']=df.apply(lambda x: x['age']+3,axis=1)` See https://www.analyticsvidhya.com/blog/2020/03/what-are-lambda-functions-in-python/